substratevm/src/com.oracle.objectfile/src/com/oracle/objectfile/debugentry/MethodEntry.java

```diff
@@ -26,18 +26,27 @@
 
 package com.oracle.objectfile.debugentry;
 
+import com.oracle.objectfile.debuginfo.DebugInfoProvider.DebugRangeInfo;
+
 public class MethodEntry extends MemberEntry implements Comparable<MethodEntry> {
     final TypeEntry[] paramTypes;
     final String[] paramNames;
     final boolean isDeoptTarget;
+    boolean isInRange;
```

Comment:
Any reason why `isInRange` should not be also final?

For final primitive fields it often makes sense to just have them public and not have an accessor method. I.e. make `isDeoptTarget` and `isInRange` public final and remove the accessor method boilerplate.

Reply:
Resolved in https://github.com/oracle/graal/pull/3419/commits/d73b4e74d453840fe0c20c018045702ce880aeab